pkg/filter/sql_event_filter.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filter
+
+import (
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	bf "github.com/pingcap/tidb-tools/pkg/binlog-filter"
```

Comment:
Better to use the full name since`bf and tfilter `is a little confused.

Reply:
IMO, the full name is too long, it is more convenient to use a short name.